analytics/analytics.go

```diff
@@ -107,13 +112,40 @@ func Initialize(n *core.IpfsNode, BTFSVersion string) {
 		dc.ArchType = runtime.GOARCH
 	}
 
-	go dc.collectionAgent()
+	go dc.collectionAgent(env)
 }
 
-func (dc *dataCollection) update() {
+func (dc *dataCollection) update(env cmds.Environment) {
 	var m runtime.MemStats
 	runtime.ReadMemStats(&m)
 
+	// get config info
+	var log = logging.Logger("cmd/btfs")
+	node, err := cmdenv.GetNode(env)
+	if err != nil {
+		log.Warning(err.Error())
+		return
+	}
+	rds := node.Repo.Datastore()
+	selfKey := ds.NewKey(fmt.Sprintf("%s%s", "/host_storage/", node.Identity.Pretty()))
```

Comment:
Make key helper a shared function, `core/commands/storage.go` also used this

Reply:
will do

Reply:
will do